src/EVTUI/Core/FileIO/Formats/EVT.cs

```diff
@@ -0,0 +1,490 @@
+                rw.RwInt32(ref this.UNK_INT32[0]);        // is a float in EvtTool but that makes no sense tbqh, also inconsistent with MAB_
+                rw.RwInt32(ref this.LoopBool);            // it's only ever 0, 1, or -1020883212 (??), so it may be a bool... LoopBool?
+                rw.RwFloat32(ref this.AnimationSpeed);    // most common is 1.0
+                rw.RwInt32(ref this.FirstFrameInd);       // (an educated guess)
+                rw.RwInt32(ref this.LastFrameInd);        // (an educated guess)
+                for (var i=0; i<4; i++)
+                    rw.RwInt32(ref this.UNUSED_INT32[i]); // observed values: 0
+            }
+
+            public void Write(string filepath) { TraitMethods.Write(this, filepath); }
+            public void Read (string filepath) { TraitMethods.Read (this, filepath); }
+        }
+
+        public class Msg_ : ISerializable
+        {
+            public Int32 MessageMode;
+            public Int16 MessageMajorId;
+            public byte MessageMinorId;
+            public byte MessageSubId;
+            public Int16 SelectMajorId;
+            public byte SelectMinorId;
+            public byte SelectSubId;
+            public Int32 EvtLocalDataIdSelStorage;
+
+            public int NumOfEntries;
+            public struct Entry
+            {
+                public Int16 UNK_INT16_1;
+                public Int16 UNK_INT16_2;
+                public float UNK_FLOAT;
+            }
+            public Entry[] Entries = new Entry[16];
+
+            public float[] UNK_FLOAT = new float[4];
+            public Int16[] UNK_INT16 = new Int16[2];
+            public Int32[] UNK_INT32 = new Int32[3];
+
+            public void ExbipHook<T>(T rw) where T : struct, IBaseBinaryTarget
+            {
+                rw.RwInt32(ref this.MessageMode);    // a bit field! that'll be handled by the manager, not here
+                rw.RwInt16(ref this.MessageMajorId);
+                rw.RwUInt8(ref this.MessageMinorId);
+                rw.RwUInt8(ref this.MessageSubId);
+                rw.RwInt16(ref this.SelectMajorId);
+                rw.RwUInt8(ref this.SelectMinorId);
+                rw.RwUInt8(ref this.SelectSubId);
+                rw.RwInt32(ref this.EvtLocalDataIdSelStorage);
+                rw.RwInt32(ref this.UNK_INT32[0]);   // observed values: 0, 1, 2, 3, 4, 5 ... number of choices to display...? but why can it be zero...
+                rw.RwFloat32(ref this.UNK_FLOAT[0]); // a very limited set
+                rw.RwFloat32(ref this.UNK_FLOAT[1]); // also a very limited set
+                rw.RwFloat32(ref this.UNK_FLOAT[2]); // huge range
+                rw.RwInt16(ref this.UNK_INT16[0]);   // seems like a set of flags (bit field?), same as next
+                rw.RwInt16(ref this.UNK_INT16[1]);   // seems like a set of flags (bit field?), same as prev
+                rw.RwInt32(ref this.UNK_INT32[1]);   // observed values: -65526, 0, 3, 7
```

Comment:
`-65526` is 0xFFFF000A, so this might be two int16s?

Reply:
Hmmm, could well be. I'm mainly getting the fields from EVTTool, and I've found a few things like this where it makes sense to diverge.

Reply:
...I think I'd rather do some experimenting before changing it, so maybe not for this PR, but worth looking into.

Reply:
(Because it's just as possible that the values in here are simply a mess....)